app/apollo/apollo-octopus-public/build.gradle.kts

```diff
@@ -36,7 +36,9 @@ apollo { // Octopus client
 
     // https://www.apollographql.com/docs/android/advanced/operation-variables/#make-nullable-variables-non-optional
     generateOptionalOperationVariables.set(false)
-
+    outputDirConnection {
+      connectToKotlinSourceSet("main") // main is by default but setting this explicitly fixed the warning "Duplicate content roots detected.
```

Comment:
Or not! Still have the warning, but that might be cache.

Reply:
It's okay to keep this then since it shouldn't be changing anything in practice